ppr-ui/src/store/actions/actions-model.ts

```diff
@@ -56,14 +57,14 @@ export const setOriginalAddCollateral: ActionIF = ({ commit }, addCollateral: Ad
   commit('mutateOriginalAddCollateral', addCollateral)
 }
 
-export const setUserAccessRole: ActionIF = ({ commit }, authRoles: Array<string>): void => {
+export const setUserAccessRole: ActionIF = ({ commit }, auth: AuthorizationIF ): void => {
   const accessRole = []
 
-  if (authRoles.includes('staff')) accessRole.push('STAFF')
+  if (auth.authRoles.includes('staff') || auth.isSbc) accessRole.push('STAFF')
   else accessRole.push('CLIENT')
 
-  if (authRoles.includes('ppr')) accessRole.push('PPR')
-  if (authRoles.includes('mhr')) accessRole.push('MHR')
+  if (auth.authRoles.includes('ppr')) accessRole.push('PPR')
+  if (auth.authRoles.includes('mhr')) accessRole.push('MHR')
   commit('mutateUserAccessRole', accessRole.join('_'))
```

Comment:
In state getters line 276, we have a computed value which identifies `isRoleStaffSbc`. 
Is that no longer required by utilizing the same flag and adding it into this array?

If that computed value is no long used in the App could we remove it?
If it is being used, we might need to look closer, as we don't want to have two means of identifying the same role/condition.

Reply:
That computed property is used throughout the app. 

In this case, I'm only passing down an SBC boolean flag, and not reusing it. I didn't want to call a getter in a setter file. Open for a discussion.

Reply:
For sure, i agree with the latter. 
The question is more with regards to using the getter itself than adding it to a reconstructed roles array. 

It feels like a duplication if we already have a global computed property indicating said role, which is something we'd like to avoid if possible.

If there is a good reason, i'm all for it. Was just curious the reason why we pass it down vs using the pre existing flag?

Reply:
Cleaned up.